pkg/sdk/grants.go

```diff
@@ -250,3 +252,32 @@ func (row grantRow) convert() *Grant {
 		GrantedBy:   NewAccountObjectIdentifier(row.GrantedBy),
 	}
 }
+
+// GrantOwnershipOptions is based on https://docs.snowflake.com/en/sql-reference/sql/grant-ownership#syntax.
+type GrantOwnershipOptions struct {
+	grantOwnership bool                    `ddl:"static" sql:"GRANT OWNERSHIP"`
+	On             OwnershipGrantOn        `ddl:"keyword" sql:"ON"`
+	To             OwnershipGrantTo        `ddl:"keyword" sql:"TO"`
+	CurrentGrants  *OwnershipCurrentGrants `ddl:"-"`
+}
+
+type OwnershipGrantOn struct {
+	SchemaObject GrantOnSchemaObject `ddl:"-"`
```

Comment:
grant ownership could also be given to account level objects like warehouses and users, not just schema objects

Reply:
Not according to the doc which is a base for this change: https://docs.snowflake.com/en/sql-reference/sql/grant-ownership: `Transfers ownership of an object or all objects of a specified type in a schema from one role to another role. Role refers to either a role or a database role.`, and also from syntax.

Reply:
Added.